readme.md

```diff
@@ -49,6 +55,13 @@ Default: `false`
 
 Only match an exact string. Useful with `RegExp#test` to check if a string is a URL.
 
+##### strict
+
+Type: `boolean`<br>
+Default: `true`
+
+Force URLs to start with a valid protocol or `www`. If set to `false` it'll match the TLD against a list of valid [TLDs](https://github.com/stephenmathieson/node-tlds).
```

Comment:
Okay, I see why it matches the list of TLDs when setting `strict` to `false`... It's because otherwise things like `foo.bar` would be considered a valid URL as well. Didn't get the reason behind it at first. LGTM!